dataprofiler/tests/profilers/test_categorical_column_profile.py

```diff
@@ -124,25 +121,22 @@ def test_categorical_mapping(self):
         num_nan_count = 2
         categories = pd.concat([df1, df2]).apply(str).unique().tolist()
         column_profile.update_profile(df2)
+        cat_profiler = column_profile.profiles['data_stats_profile']._profiles["category"]
```

Comment:
is cat_profiler generated from the beginning of this test? If it's regenerated, does it change the test meaning?

Reply:
Previously the test had a bug, because it assumed the object reference was consistent. Since we switched to multiprocessing the object is no longer going to have the same reference. Because of that we have to see the current properties of the class.